xmipptomo/protocols/protocol_detect_landmarks_TS.py

```diff
@@ -0,0 +1,234 @@
+# **************************************************************************
+# *
+# * Authors:     Federico P. de Isidro Gomez (fp.deisidro@cnb.csi.es) [1]
+# *
+# * [1] Centro Nacional de Biotecnologia, CSIC, Spain
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+
+import os
+from pyworkflow import BETA
+import pyworkflow.protocol.params as params
+import pyworkflow.utils.path as path
+from pyworkflow.object import Set, Boolean
+from pyworkflow.protocol.constants import STEPS_PARALLEL
+from pwem.protocols import EMProtocol
+import tomo.objects as tomoObj
+from tomo.protocols import ProtTomoBase
+import xmipptomo.utils as utils
+
+
+OUTPUT_COORDS_FILENAME = "outputLandmarkCoordinates.xmd"
+
+
+class XmippProtDetectLandmarkTS(EMProtocol, ProtTomoBase):
+    """
+    Scipion protocol for xmipp_tomo_detect_landmarks. Detect landmarks in a tilt series.
```

Comment:
add more help to explain what the protocol does, including key words to simplify the search of the protocol in the search box of scipion. For instance, including the word gold bead or fiducial

Reply:
Done